lib/Echidna/Types/Tx.hs

```diff
@@ -23,6 +23,18 @@ data TxCall = SolCreate   ByteString
 makePrisms ''TxCall
 $(deriveJSON defaultOptions ''TxCall)
 
+maxGasPerBlock :: Integer
+maxGasPerBlock = 12500000 
```

Comment:
any reason for 8000030 ->12500000 besides a less random number?

Reply:
This was recently [increased by miners](https://cointelegraph.com/news/ethereum-miners-vote-to-increase-gas-limit-causing-community-debate).